server/hedley/modules/custom/hedley_stats/hedley_stats.module

```diff
@@ -3802,3 +3809,16 @@ function hedley_stats_add_patient_details($node, array &$patients_details, $add_
     }
   }
 }
+
+/**
+ * Extracts the first value from a comma-separated string of values.
+ *
+ * @param string $values_string
+ *   The values string.
+ *
+ * @return string
+ *   First value.
+ */
+function hedley_stats_extract_first_value($values_string) {
+  return explode(',', $values_string)[0];
```

Comment:
Should we play safely? So I am not sure for instance `explode` will return an array in every case.

Reply:
![Selection_529](https://github.com/TIP-Global-Health/eheza-app/assets/17882245/cd54cd45-f9b9-4d82-a4b6-5b0243d3be92)
![Selection_527](https://github.com/TIP-Global-Health/eheza-app/assets/17882245/ebffc6d8-68c6-41e5-a71b-6d395605298d)
![Selection_528](https://github.com/TIP-Global-Health/eheza-app/assets/17882245/ec127756-c18d-46f2-baed-1a2f5188bf7e)